packages/core/src/provider/base.ts

```diff
@@ -405,7 +405,7 @@ export class SmartAccountProvider<
       // this pretty prints the uo
       throw new Error(
         `Request is missing parameters. All properties on UserOperationStruct must be set. uo: ${JSON.stringify(
-          request,
+          uoStruct,
```

Comment:
was there a reason not to use `request` here?

Reply:
thought hex is less readable, because users will identify numbers better.